packs/qualys/actions/list_scans.py

```diff
@@ -0,0 +1,13 @@
+from lib.base import QualysBaseAction
+
+__all__ = [
+    'ListScansAction'
+]
+
+
+class ListScansAction(QualysBaseAction):
+    def run(self, launched_after="", state="", target="",
```

Comment:
Usually it's a better idea to default to `None` for optional arguments and then if the client requires a string handle that when calling a client method.